tests/integration/test_storage_s3/test.py

```diff
@@ -278,3 +279,30 @@ def test_wrong_s3_syntax(cluster, s3_storage_args):
 
     query = "create table test_table_s3_syntax (id UInt32) ENGINE = S3({})".format(s3_storage_args)
     assert expected_err_msg in instance.query_and_get_error(query)
+
+
+def test_s3_glob_scheherazade(cluster):
```

Comment:
Need a comment with cultural references. Not everyone is familiar what is "scheherazade". Wikipedia Ok.